server/controllers/users/login.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable consistent-return */
+const bcrypt = require('bcrypt');
+const { loginSchema } = require('../../utils/validation/loginSchema');
+const { checkEmailQuery } = require('../../database/quieres');
+const { signToken } = require('../../utils');
+
+const login = async (req, res, next) => {
+  try {
+    const { email, password } = req.body;
+    await loginSchema.validateAsync(req.body);
+
+    const { rows } = await checkEmailQuery(email);
+
+    if (!rows.length) {
+      throw new Error({ message: 'Invalid email or password' });
+    }
+    const compared = await bcrypt.compare(password, rows[0].password);
```

Comment:
You can make your controller a little bit organizer by splitting these
```suggestion
    }
    
    const compared = await bcrypt.compare(password, rows[0].password);
```